app/graphql/types/query_type.rb

```diff
@@ -49,6 +50,7 @@ def aid_requests(city:, state:)
       AidRequest.joins(:organization).where("city ILIKE ? AND state ILIKE ?", "%#{city}%", "%#{state}%")
     end
 
+    # All locations sorted by state, alphabetically
```

Comment:
So a refactor we could consider here is adding these comments to a `description:` field like this:
`field :aid_requests, [Types::AidRequestType], null: false, description: "AidRequests for a given city+state" do`